### DISCUSSION TITLE
[SUGGESTION] Mob Turn-Pathing & Bounty $250

### DISCUSSION THREAD

**Discussion by heyaapl** [azerothcore/azerothcore-wotlk#3938] [Ideas]
I would really love a smooth-turn of mobs when pathing. Today, they hard-snap turn to directly face their next waypoint. Is anyone aware of a module that would enable a smooth-turn when an NPC is pathing (both waypoints and random-pathing for world creatures)?

This is something that exists in WoW Classic, and retail WoW, and I feel like this would add a level of polish to AzerothCore that I'd love to see. Be this a core enhancement that is configurable (for performance purposes) or a module. 

I'm willing to help fund this up to $250.

**Comment by FrancescoBorzi** [comment]
If this is blizzlike behaviour, the issue can be moved in the main repo and a bounty can be placed directly there.

What are the opinions about this?

**Comment by SinCosAtan** [comment]
Contact me on discord @heyaapl i have what you seek for.

Discord: ProGamer123#1849
